app/src/main/java/eu/kanade/tachiyomi/data/updater/AppUpdateChecker.kt

```diff
@@ -45,6 +45,8 @@ class AppUpdateChecker {
 val GITHUB_REPO: String by lazy {
     if (BuildConfig.PREVIEW) {
         "mihonapp/mihon-preview"
+    } else if (BuildConfig.FLAVOR == "foss") {
+        "mihonapp/mihon-foss"
```

Comment:
@AntsyLich Let me know if I need to change the repo name

Reply:
@AntsyLich Do you want me to initialize a repo you will then fork? Let me know if I can help ;)

Reply:
Kinda busy and this is low priority. I'll get it merged soon